Cargo.toml

```diff
@@ -279,6 +276,7 @@ name = "ring"
 [dependencies]
 libc = "0.2.34"
 untrusted = "0.6.1"
+native_versioning = { git = "https://github.com/SergioBenitez/native_versioning" }
```

Comment:
In order for us to be able to use this it would need a MIT/ISC/Apache2 license and it would need to be released on crates.io.

Reply:
Of course. I've added the license to the repository, and as I said in the PR, I'll release it on crates.io once this PR is ready to be merged.

Reply:
I don't want to use the crate w/o it also being ISC licensed. I don't have any intention of forking it or folding it into *ring*, but I also want the reassurance that we could fold a variant of it into *ring* if the dependency were to ever become problematic for any reason, and that would only be realistic if it were ISC licensed since the rest of the *ring* build is ISC licensed.

Reply:
I really don't want to get into a license discussion here, but why isn't the MIT license sufficient? It would clearly allow you to do what you're proposing.

Reply:
> I really don't want to get into a license discussion here, but why isn't the MIT license sufficient? It would clearly allow you to do what you're proposing.

I also want to avoid a licensing discussion. *ring*'s build.rs is ISC-licensed and we need to be able to deal with unwanted changes to the native_versioning crate by being able to copy the code into *ring* using the same license we use for all the Rust code in *ring*. I don't want to be in the situation where our only reasonable alternative would be to do a clean-room implementation of the logic since the logic. Anyway, it's understandable if you don't want to add the ISC license option to your crate; in that case, please just close this PR.

Reply:
I think it's pretty reasonable to accept the MIT license in `native_versioning`. But, I really don't care, so I've added the ISC license to `native_versioning`.

Reply:
I think it's rather reasonable to accept the MIT license. But, I really don't care to argue this further, so I've added the ISC license to `native_versioning`.